example_site/src/components/VideoInfo.js

```diff
@@ -1,26 +1,65 @@
 import "./VideoInfo.css"
+import { months, days } from "./constants";
 
 /**
  * Renders a video component showing video thumbnail and information
  * 
  * @param {Object} video Video object
  */
-const VideoInfo = ({ video }) => {
-    const createdDate = (video.createdAt + "").substr(0, 15);
-    const updatedDate = (video.updatedAt + "").substr(0, 15);
+
+function getThumbnailWidth(url) {
+    var img = new Image();
+    img.src = url;
+    return img.width;
+}
+
+function getThumbnailHeight(url) {
+    var img = new Image();
+    img.src = url;
+    return img.height;
+}
+
+function getMonthName(videoDate){
+    const monthName = months[videoDate.getMonth()];
+    return monthName;
+}
+
+function getWeekdayName(videoDate){
+    const dayName = days[videoDate.getDay()];
+    return dayName;
+}
```

Comment:
These are nice and generic, let's make them utility function that we can import! And I'm just now realizing that we should move our constants file :) 

Let's update the `example_site` directory to look like this:
```
example_site/
    src/
        components/
        public/
        constants/
            app.js
        utils/
            date.js
```

So you'll move `example_site/src/components/constant.js` to `example_site/src/constants/app.js` and update all of the imports

And then move your `months`, and `days` constants to `example_site/src/utils/date.js` along with the `getMonthName` and `getWeekdayName` functions and just import the functions you need into `VideoInfo.js`.

Additionally, make sure there is a space between `){` in your function declarations!

Reply:
And make sure to put comments above the functions explaining what they're doing!